### DISCUSSION TITLE
[A-L] (9 Settembre 2021) Esercizio 10

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#299] [Esami A-L]
![image](https://user-images.githubusercontent.com/69389461/152793995-951c765e-0e1c-40f0-b01b-b290909f891e.png)

**Comment by LuginiAndrea** [comment]
Intepretazione pari ( $A$ è l'insieme sul quale è definita $R$ ):

1. Ignorando la definizione di $R$, interpretiamo come  $$\exists y \in A \textrm{ t.c. } \forall x \in A \textrm{ } xRy \implies \forall x \in A \textrm{ } \exists y \in A \textrm{ t.c. } xRy$$ Facile vedere come si tratti di una tutologia
2. Definiamo $R$ come $xRy \iff x = y$, ed interpretiamo come $$\forall x \in A, \forall y \in A, \left(xRy  \implies x \neq y\right)$$ Immediato come ciò sia insoddisfacibile per definizione stessa di $R$
